package.json

```diff
@@ -36,6 +36,7 @@
   "dependencies": {
     "appium-android-driver": "1.x",
     "appium-base-driver": "2.x",
+    "appium-espresso-driver": "^1.0.0-beta.1",
```

Comment:
How does npm semver work with weird version "numbers". We could, instead, just specify "beta" to get whatever is most recently tagged as beta.